docs/organizer-guide/confs/community-events.rst

```diff
@@ -0,0 +1,20 @@
+Community Events
+================
+
+We are happy to promote events that are created by our community.
```

Comment:
created by our community > independently organized by members of our community

Reply:
This is an arrow, not a greater than? :+1: